test/extended/olm/olmv1.go

```diff
@@ -99,6 +101,43 @@ var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLM
 	})
 })
 
+var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLMv1 Catalog's /v1/api/all endpoint", func() {
```

Comment:
I'd suggest removing the quotation marks or anything that might complicate searching for these strings in https://search.dptools.openshift.org/